docs/runbooks/VirtHandlerRESTErrorsHigh.md

```diff
@@ -2,17 +2,38 @@
 
 ## Meaning
 
-More than 5% of the rest calls failed in virt-handler for the last hour
+More than 5% of REST calls failed in `virt-handler` in the last 60 minutes. This is most likely because `virt-handler` has partially lost connection to the apiserver.
 
 ## Impact
 
-Virt-handler has partially lost the connection to the apiserver. Node-related actions (where virt-handler is running on) like starting and migrating workloads will be delayed. Running workloads will not be affected, but reporting their current status may be delayed.
+Node-related actions, such as starting and migrating workloads, will be delayed on the node that `virt-handler` is running on. Running workloads will not be affected, but reporting their current status may be delayed.
```

Comment:
I recommend changing these sentences to present tense instead of future. [IGS](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense).